Curti/template/simple_functions.py

```diff
@@ -0,0 +1,29 @@
+def fibonacci(max):
+    values = [0, 1]
+    while values[-2] + values[-1] < max:
+        values.append(values[-2] + values[-1])
+    return values
+
+
+def factorial(value):
+    if value == 0:
+        return 1
+    else:
+        return value * factorial(value - 1)
+
+
+def is_prime(a) : 
+
```

Comment:
what if a is negative? your function will still return true

Reply:
should be ok now.
Thank you!